### DISCUSSION TITLE
[@types/node] URL and URLSearchParams are not exposed in the global scope

### DISCUSSION THREAD

**Discussion by dd-pardal** [DefinitelyTyped/DefinitelyTyped#57736] [Issues with a @types package] [answered]
What the title says. Both `URL` and `URLSearchParams` are available in the global scope in Node.js but TypeScript doesn't know they are.

(Note: This sometimes goes unnoticed because the `dom` library is included by default when `compilerOptions.lib` isn't specified in `tsconfig.json` and they are declared globally there.)

[a comment by an automated account is not shown]

**Comment by thw0rted** [comment, marked as answer]
If you're still tracking this @dd-pardal could you please test again?  I just looked at the types for URL /  URLSearchParams and they appear to be exposed globally now.

**Reply by dd-pardal** [reply]
> If you're still tracking this @dd-pardal could you please test again? I just looked at the types for URL / URLSearchParams and they appear to be exposed globally now.

I've just tested it with the latest version (`14.18.13`) of `@types/node` and it's working as expected. Thank you.
